utils_cv/detection/plot.py

```diff
@@ -86,32 +89,55 @@ def plot_boxes(
     return im
 
 
-def display_bboxes(
-    bboxes: List[_Bbox],
-    im_path: Union[Path, str],
-    ax: Union[None, plt.axes] = None,
+def plot_mask(
+    im: Union[str, Path, PIL.Image.Image],
+    mask: Union[str, Path, np.ndarray],
     plot_settings: PlotSettings = PlotSettings(),
+) -> PIL.Image.Image:
+    """ Put mask onto image.
+
+    Assume the mask is already binary masks of [N, Height, Width], or
+    grayscale mask of [Height, Width] with different values
+    representing different objects, 0 as background.
+    """
+    if isinstance(im, (str, Path)):
+        im = Image.open(im)
+
+    # convert to RGBA for transparentising
+    im = im.convert('RGBA')
+    # colorise masks
+    binary_masks = binarise_mask(mask)
+    colored_masks = [
+        colorise_binary_mask(bmask, plot_settings.mask_color) for bmask in
+        binary_masks
+    ]
+    # merge masks into img one by one
+    for cmask in colored_masks:
+        tmask = Image.fromarray(
+            transparentise_mask(cmask, plot_settings.mask_alpha)
+        )
+        im = Image.alpha_composite(im, tmask)
+
+    return im
+
+
+def display_image(
```

Comment:
If only used once, then please delete this function and move code.

Reply:
Removed